test/crux/fixtures.clj

```diff
@@ -1,28 +1,45 @@
 (ns crux.fixtures
-  (:require [clojure.java.io :as io]
-            [clojure.spec.alpha :as s]
+  (:require [clojure.spec.alpha :as s]
             [clojure.test :as t]
             [clojure.test.check.clojure-test :as tcct]
-            [crux.api :as api]
+            [clojure.java.io :as io]
             [crux.bootstrap :as b]
             [crux.codec :as c]
             [crux.db :as db]
             [crux.http-server :as srv]
-            [crux.index :as idx]
             [crux.io :as cio]
-            [crux.kafka :as k]
             [crux.kafka.embedded :as ek]
+            [crux.kafka :as k]
             [crux.kv :as kv]
-            [crux.lru :as lru]
-            [crux.memory :as mem]
             [crux.tx :as tx]
-            [taoensso.nippy :as nippy])
-  (:import [crux.api Crux ICruxAPI]
-           java.io.Closeable
+            [crux.index :as idx]
+            [crux.lru :as lru]
+            [taoensso.nippy :as nippy]
+            [crux.memory :as mem])
+  (:import java.io.Closeable
            [java.util Properties UUID]
            org.apache.kafka.clients.admin.AdminClient
+           org.apache.kafka.clients.producer.KafkaProducer
            org.apache.kafka.clients.consumer.KafkaConsumer
-           org.apache.kafka.clients.producer.KafkaProducer))
+           [crux.api Crux ICruxAPI]))
+
+(defn random-person [] {:crux.db/id (UUID/randomUUID)
```

Comment:
I moved this people stuff to the bottom of this ns, make sure we don't duplicate

Reply:
sorry, I meant to see the diff here and revert it